docs/resources/job.md

```diff
@@ -69,6 +69,7 @@ The following arguments are required:
 
 * `quartz_cron_expression` - (Required) A [Cron expression using Quartz syntax](http://www.quartz-scheduler.org/documentation/quartz-2.3.0/tutorials/crontrigger.html) that describes the schedule for a job. This field is required.
 * `timezone_id` - (Required) A Java timezone ID. The schedule for a job will be resolved with respect to this timezone. See Java TimeZone for details. This field is required.
+* `pause_status` - (Optional) Indicate whether this schedule is paused or not. Either “PAUSED” or “UNPAUSED”.
```

Comment:
maybe write `UNPAUSED (default)`?

Reply:
is it default, though?

Reply:
hmm... @alexott i dont think we can provide a default if the field itself is computed. So in theory we dont provide a value when we make the rest call. So it is what ever our rest api chooses to do when that field is not omitted. And our rest api docs dont seem to indicate that there is a default. Should I check with them to verify if there is a built in default?

Reply:
maybe I incorrectly formulated - we need to say that this specific will be used if it's omitted. Of course it's enforced on API level... but that was just a comment

Reply:
ahh gotcha i can add the create behavior if it is omitted.

Reply:
added this: `When the pause_status field is omitted and a schedule is provided, the server will default to using "UNPAUSED" as a value for pause_status.`